assets/src/edit-story/app/story/actions/useSavePost.js

```diff
@@ -29,8 +31,9 @@ const getStoryMarkupFromPages = ( pages ) => {
 						return <Save key={ 'element-' + elId } { ...rest } />;
 					} ) }
 				</amp-story-grid-layer>
-			</amp-story-page>,
+			</amp-story-page>
```

Comment:
Does this actually achieve the desired result? Isn't the problem the `markup.join()` which default the join delimiter to a comma? So `markup.join( '' )` would be the proper fix?

I can't think that this dangling comma is ever sent to the underlying function - it's purely syntax (and I even expect babel to optimize it away).

Reply:
Yep, you're right, completely missed the `markup.join` using the default comma... Will create another (real) fix for this.

Reply:
https://github.com/ampproject/amp-wp/pull/4027